test/e2e/specs/publishing.test.js

```diff
@@ -4,48 +4,53 @@
 import {
 	newPost,
 	publishPost,
+	publishPostWithoutPrePublishChecks,
+	disablePrePublishChecks,
 } from '../support/utils';
 
 describe( 'Publishing', () => {
-	describe( 'a post', () => {
-		beforeEach( async () => {
-			await newPost();
-		} );
+	[ 'post', 'page' ].forEach( ( postType ) => {
+		describe( `a ${ postType }`, () => {
+			beforeEach( async () => {
+				await newPost( postType );
+			} );
 
-		it( 'should publish a post and close the panel once we start editing again', async () => {
-			await page.type( '.editor-post-title__input', 'E2E Test Post' );
+			it( `should publish the ${ postType } and close the panel once we start editing again.`, async () => {
+				await page.type( '.editor-post-title__input', 'E2E Test Post' );
 
-			await publishPost();
+				await publishPost();
 
-			// The post-publishing panel is visible.
-			expect( await page.$( '.editor-post-publish-panel' ) ).not.toBeNull();
+				// The post-publishing panel is visible.
+				expect( await page.$( '.editor-post-publish-panel' ) ).not.toBeNull();
 
-			// Start editing again.
-			await page.type( '.editor-post-title__input', ' (Updated)' );
+				// Start editing again.
+				await page.type( '.editor-post-title__input', ' (Updated)' );
 
-			// The post-publishing panel is not visible anymore.
-			expect( await page.$( '.editor-post-publish-panel' ) ).toBeNull();
+				// The post-publishing panel is not visible anymore.
+				expect( await page.$( '.editor-post-publish-panel' ) ).toBeNull();
+			} );
 		} );
 	} );
 
-	describe( 'a page', () => {
-		beforeEach( async () => {
-			await newPost( 'page' );
-		} );
-
-		it( 'should publish a page and close the panel once we start editing again', async () => {
-			await page.type( '.editor-post-title__input', 'E2E Test Page' );
+	[ 'post', 'page' ].forEach( ( postType ) => {
+		describe( `a ${ postType } with pre-publish checks disabled`, () => {
+			beforeEach( async () => {
+				await newPost( postType );
+				await disablePrePublishChecks();
```

Comment:
Because this setting persists between page loads we should be careful with this… arguably we should have some kind of consistent/known state and after each test where we disable the pre-publish check we should re-enable them with the `afterEach` test helper.

Right now this test will disable pre-publish checks elsewhere which–as it's not the default state–could be confusing to future test writers.

Reply:
Done, good catch!